docs/topics/home.xml

```diff
@@ -39,9 +39,9 @@
             </group>
             <group type="cards">
                 <title>Kotlin Multiplatform Mobile (KMM)</title>
-                <a href="kmm-getting-started.md" description="A simple mobile cross-platform application">Get started with KMM in Android Studio</a>
-                <a href="kmm-plugin-releases.md" description="Features of the KMM plugin">KMM plugin releases for Android Studio</a>
-                <a href="https://www.youtube.com/playlist?list=PLlFc5cFwUnmy_oVc9YQzjasSNoAk4hk_C" description="Videos about KMM on Kotlin YouTube channel">Kotlin Multiplatform Multiverse</a>
+                <a href="kmm-getting-started.md" description="A simple cross-platform mobile application">Get started with KMM in Android Studio</a>
```

Comment:
Are we leaving "KMM" in the tutorial name?

Reply:
Yep.
I rollbacked other abbreviations 'cause they don't fit their boxes.
Need to think how to rephrase them all.